dist/obs-rpmlintrc

```diff
@@ -8,3 +8,4 @@ addFilter('zero-length')
 addFilter('files-duplicate')
 addFilter('script-without-shebang')
 addFilter('devel-file-in-non-devel-package')
+addFilter('unexpanded-macro')
```

Comment:
The commit message is wrong, it says ``Rpmlint: Silence devel-file-in-non-devel-package warning`` but you actually add the ``unexpanded-macro`` rpmlintrc fiter.